clients/deltalake/src/test/java/org/projectnessie/deltalake/ITDeltaLog.java

```diff
@@ -29,19 +29,25 @@
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.functions;
 import org.apache.spark.util.Utils;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
 import org.junit.jupiter.api.io.TempDir;
+import org.projectnessie.client.api.NessieApiV1;
+import org.projectnessie.client.api.NessieApiVersion;
+import org.projectnessie.client.http.HttpClientBuilder;
 import org.projectnessie.client.tests.AbstractSparkTest;
 import org.projectnessie.model.Branch;
-import org.projectnessie.model.ImmutableMerge;
 import org.projectnessie.model.Reference;
 import scala.Tuple2;
 
 class ITDeltaLog extends AbstractSparkTest {
 
+  NessieApiV1 api;
```

Comment:
nit: `private`?

Reply:
nah - it's a test

Reply:
BTW: that's code from #1932 (which is why this PR is in draft)